src/vue/align-editor/align-editor-single-block.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+    <div class  ="alpheios-alignment-editor-align-text"
+         :class = "alignTextClass" 
+         :dir = "direction" :lang = "lang" 
+         @click = "clickEmptyText"
+    >
+      <template v-for = "token in alignTextData.tokens">

```

Comment:
no, it is not right. If you would add :key to the v-for inside template tag - it will be broken and won't be rendered.
I believe, that's why you didn't add it to `WideInflectionsTable` in components to template tag.

Reply:
Oh, you're right, I've missed that it's a template. The templates cannot be keyed. It is recommended, however, to put keys for elements within the template, see the discussion here: https://forum.vuejs.org/t/v-for-on-templates/8359. I'm not sure I was following these practices everywhere, but I think I should.

Reply:
thank you for this information, added to next step!